spec/models/tag_group_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+describe TagGroup, type: :model do
+
+  describe "#bg_color" do
+
+    context "with valid json" do
+      let(:valid_json) { "{\"r\":\"241\",\"g\":\"112\",\"b\":\"19\",\"a\":\"1\"}" }
```

Comment:
Kinda hard to read this...

If you use single quote, the backslashes will not be necessary anymore. Or, you could do
```ruby
 let(:valid_json) { { r: 241, g: 112, b: 19, a: 1 }.to_json }
```